charts/minecraft-bedrock/templates/datadir-pvc.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     app: {{ template "minecraft.fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}"
```

Comment:
A template function would be good.  I know it doesn't help tremendously since it'll be per chart, but still would be nice to encapsulate the gory bits.